packages/design-tokens/package.json

```diff
@@ -33,7 +33,9 @@
     "test": "jest"
   },
   "devDependencies": {
-    "@primer/primitives": "9.0.2",
+    "@primer/primitives": "9.1.0",
+    "prettier": "^2.8.8",
```

Comment:
We should already have this available in the root for all workspaces: https://github.com/primer/brand/blob/main/package.json#L68

You can keep `@prettier/sync` though, could that be a dependency of `primer/primitives` instead? Reason being that I'm not seeing any direct imports to that library, which indicates that it's an 3rd party dependency.

Reply:
https://github.com/github/primer/issues/3903